backend/Dockerfile

```diff
@@ -5,5 +5,5 @@ RUN pip install -r requirements.txt
 
 ENTRYPOINT ["./server_setup_entrypoint"]
 
-EXPOSE 5000
+EXPOSE 80
```

Comment:
Both ports 80 and 5000 need to be exposed, so could you add an additional line?
`EXPOSE 5000`

Reply:
Done